lib/rucio/core/rse.py

```diff
@@ -662,39 +660,51 @@ def get_rses_with_attribute_value(key, value, lookup_key, vo='def', session=None
 
 
 @read_session
-def get_rse_attribute(key, rse_id=None, value=None, use_cache=True, session=None):
+def get_rse_attribute_without_cache(rse_id: str, key: str, session: Optional[Session] = None) -> Optional[Union[str, bool]]:
     """
-    Retrieve RSE attribute value.
+    Retrieve RSE attribute value from the database.
 
     :param rse_id: The RSE id.
     :param key: The key for the attribute.
-    :param value: Optionally, the desired value for the attribute.
-    :param use_cache: Boolean to use memcached.
     :param session: The database session in use.
 
-    :returns: A list with RSE attribute values for a Key.
+    :returns: The value for the rse attribute, None if it does not exist.
     """
 
-    result = NO_VALUE
-    if use_cache:
-        result = REGION.get('%s-%s-%s' % (key, rse_id, value))
-    if result is NO_VALUE:
+    value = session.query(models.RSEAttrAssociation.value).filter_by(rse_id=rse_id, key=key).first()
```

Comment:
[`Query.one_or_none`](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.one_or_none) could be useful here instead of `Query.first`, depends on how you'd like to handle exceptions. This is supposed to be a primary key after all.

Reply:
I like it. Let's not handle the exception since it's a primary key.